src/components/Dashboard/TodoList.vue

```diff
@@ -1756,7 +1758,9 @@ export default class TodoList extends Vue {
     color: $app-blue;
   }
 
-  #btn-draft-delete:hover {
+  #btn-draft-delete:hover,
+  #btn-delete-application:hover,
+  #btn-cancel-payment:hover {
     background-color: $gray1;
```

Comment:
Hover styling was missing altogether from the Cancel Payment button.

The Delete Application button had default Vuetify styling, which is a very light gray. Now it's darker gray and consistent.